cristin-commons/src/main/java/no/unit/nva/cristin/model/Constants.java

```diff
@@ -67,9 +82,153 @@ public class Constants {
     public static final String CRISTIN_BOT_FILTER_BYPASS_HEADER_VALUE =
         ENVIRONMENT.readEnv("CRISTIN_BOT_FILTER_BYPASS_HEADER_VALUE");
 
-    public enum QueryType {
-        QUERY_USING_GRANT_ID,
-        QUERY_USING_TITLE
-    }
+    private static final String PATTERN_IS_NUMBER6 = "[1-9]\\d*";
+    public static final String PATTERN_IS_DATE = "(\\d){4}-(\\d){2}-(\\d){2}[T]*[(\\d){2}:(\\d){2}:(\\d){2,6}Z]*";
+    private static final String PATTERN_IS_NON_EMPTY = ".+";
+    private static final String PATTERN_IS_TITLE = "^[æøåÆØÅ\\w-,\\. ]+$";
+    private static final String PATTERN_IS_LANGUAGE = "(en|nb|nn|\\,)+";
+    private static final String PATTERN_IS_STATUS = "(?i)CONCLUDED|ACTIVE|NOT[ +]*STARTED";
+    public static final String PATTERN_IS_URL =
+        "(http(s):\\/\\/.)[-a-zA-Z0-9@:%._\\+~#=]{2,256}\\.[a-z]{2,6}\\b([-a-zA-Z0-9@:%_\\+.~#?&//=]*)";
+
+    public enum QueryParameterKey {
```

Comment:
When this enum have become so large, it is better that it has its own enum file in the file system, not an inner enum